common/diplomatic_actions/16_bankroll.txt

```diff
@@ -0,0 +1,419 @@
+﻿bankroll = {
```

Comment:
Since the rank requirement was removed it could be nice to re-add it for the AI to prevent non-sensical bankrolls of minor to great powers for example.

Reply:
still waiting

Reply:
I dont think it's necessary